Tella/Data/Database/Vault/VaultDataBase.swift

```diff
@@ -146,7 +163,39 @@ class VaultDatabase : DataBase, VaultDataBaseProtocol {
             return nil
         }
     }
+
+    func getNonUpdatedVaultFiles() -> [VaultFileDB] {
+        do {
+            let vaultFilesDict = try statementBuilder.getSelectQuery(tableName: VaultD.tVaultFile,
+                                                                     equalCondition:[KeyValue(key: VaultD.cEncryptionUpdated, value: false)])
+            
+            let vaultFiles = vaultFilesDict.compactMap{VaultFileDB.init(dictionnary: $0)}
+            return vaultFiles
+            
+        } catch {
+            return []
```

Comment:
Let's provide more meaningful error messages for better debugging, let's not keep this epmty ?